### ISSUE TITLE
ISO's are deployed to SR instead of SR_ISO

### ISSUE THREAD

**Issue by tschilb252** [ddelnano/packer-plugin-xenserver#23]
Attached is the default template I am using to deploy a CentOS8 ISO to my SR and then build the machines.  I have 2 issues:

1. The ISO is copied to my SR and NOT my SR_ISO
2. It creates a duplicate ISO every time

  I have attached 2 screenshots of each SR I have showing the issue at hand along with the configuration file I used to deploy the CentOS 8 image.

I am using this script to execute the centos8.json file (IP, username, and password masked):

```
# Export variables for the script
echo writing variables...

export XAPI_HOST='10.xxx.xxx.xxx'
export XAPI_USERNAME='username'
export XAPI_PASSWORD='***************'

echo starting deployment...
PACKER_LOG=1 packer build -debug --var sr_name='Equallogic 10k SAS - HQ2' --var sr_iso_name='ISOs - HQ2' centos8.json
```
  
![SR_ISO showing a lack of the centos8 ISO being uploaded](https://user-images.githubusercontent.com/73147140/113888657-8a852b00-9790-11eb-9637-5a6d7e1b3a56.png)

![SR showing multiple uploads of the same ISO](https://user-images.githubusercontent.com/73147140/113888697-940e9300-9790-11eb-9c1b-f0a36c7a2dcb.png)

[centos8.txt](https://github.com/ddelnano/packer-plugin-xenserver/files/6272628/centos8.txt)

**Comment by tschilb252** [comment]
UPDATE:  only when an ISO is downloaded from a remote URL is it placed in the SR instead of the SR_ISO.

I was able to upload a local ISO file into the SR_ISO properly when the iso_url and iso_name property are the same. If I change the iso_url to an HTTP address, it uploads the ISO to the SR and not the SR_ISO.

**Comment by AtaxyaNetwork** [comment]
>What we need is a check in the GO code that determines if that name already exists in the ISO repository and then upload or skip and just reference if its there.

Already done:
https://github.com/ddelnano/packer-plugin-xenserver/commit/4f90bf5f176d8e640b7775fc079dc456e7b3e035

**Comment by ddelnano** [comment]
@AtaxyaNetwork that would be great. I finally have a more stable test environment for this plugin, which was a big hurdle for me quickly reviewing changes. Getting fully working CI would be great, but for now I will be able to turn around PR review more quickly.

**Comment by AtaxyaNetwork** [comment]
Done !

**Comment by ddelnano** [comment]
This will be fixed in v0.5.3, which should be available in the next hour or so!
